doc/RobotFirmware2015-OSX.md

```diff
@@ -0,0 +1,86 @@
+# OSX Firmware Setup
```

Comment:
Rather than having install instructions, can we add all this to the `osx-setup` script?  I think we can automate all of these things.  The `ubuntu-setup` script has a `--firmware` command line option that installs the firmware-related things.  It'd probably be good to do the same for OS X.